lib/middleware/error_handler.coffee

```diff
@@ -0,0 +1,14 @@
+path = require 'path'
+{ NODE_ENV } = require '../../config'
+
+module.exports = (err, req, res, next) ->
+  file = path.resolve(
+    __dirname,
+    '../../desktop/components/error_handler/index.jade'
+  )
+  isNotProduction = NODE_ENV isnt 'production'
+  code = 504 if req.timedout
+  code ||= err.status || 500
+  message = err.message || err.text || err.toString() if isNotProduction
+  detail = err.stack if isNotProduction
+  res.status(code).render(file, { message, detail, code })
```

Comment:
That should be caught [here](https://github.com/artsy/force/blob/master/lib/setup.js#L197), but we should move that middleware just above this one... because now that I think about it—that explains why Sentry is only reporting errors in middleware and not the apps... [noted](https://github.com/artsy/force/issues/1401).